### PR TITLE
[ES] Add `Guides/Common modding mistakes`

### PR DESCRIPTION
**Pull request by MChecaH**
Translation of [Common modding mistakes](https://github.com/ppy/osu-wiki/blob/master/wiki/Guides/Common_modding_mistakes/en.md).

First direct contribution. If anything is amiss, please let me know. Not used to the terminology that gets translated Vs. the one reused from English.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```md
+En general, las notas deberían estar mapeadas a distintos sonidos de la canción para poder reflejarla con exactitud. Mantener la densidad del ritmo con la intensidad de la canción es /.../
+```
+
+**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar iendo por las ramas, olvidándote de mejorar el mapa en el proceso.
+
+---
+
+```md
+El soporte para la resolución panorámica es inconsistente y tu mapa no tiene guin gráfico, por lo tanto desactívalo para todas las dificultades.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:77]:
```suggestion
El soporte para la resolución panorámica es inconsistente y tu mapa no tiene storyboard, por lo tanto desactívalo para todas las dificultades.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+00:00:000 - La manta está desalineada por 2 píxeles.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:78]:
```suggestion
00:00:000 - El blanket está desalineada por 2 píxeles.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+---
+
+```md
+00:00:000 - tio, ¿siempre mapeas así de mal?
+aprende a mappear y entonces podré modearlo correctamente
+este mapa se vería increible en la basura
+```
+
+**Evita ser grosero o sarcástico cuando modeas.** Esta conducta acabará dejándote [silenciado](/wiki/Silence), y no es algo que a otras personas les guste leer. Asegúrate de seguir el [Código de Conducta](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) cuando modeas para mantener un ambiente de modding positivo para todos.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:168]:
```suggestion
**Evita ser grosero o sarcástico cuando modeas.** Esta conducta acabará dejándote [silenciado](/wiki/Silence), y no es algo que a otras personas les guste leer. Asegúrate de seguir el [código de conducta](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) cuando modeas para mantener un ambiente de modding positivo para todos.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
+# Errores de modding comunes
+
+*Véase también: [Convertirse en Beatmap Nominator § Expectativas de modding](/wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator)*
+
+Esta guía enseñará varios ejemplos de errores que [modders](/wiki/Modding/Modder) principiantes suelen cometer cuando [modean](/wiki/Modding) un [beatmap](/wiki/Beatmap).
+
+Los siguientes puntos están presentados de manera que un mal ejemplo de modding se muestra arriba, y una explicación de por qué es un error abajo.
+
+Aunque los ejemplos enseñados són modelados a partir de contexto de mapping específico a [osu!](/wiki/Game_mode/osu!), también se pueden aplicar a otros [modos de juego](/wiki/Game_mode).
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:9]: catalan?
```suggestion
Aunque los ejemplos enseñados son modelados a partir de contexto de mapping específico a [osu!](/wiki/Game_mode/osu!), también se pueden aplicar a otros [modos de juego](/wiki/Game_mode).
```

File: wiki/Guides/Common_modding_mistakes/es.md (hunk 1 above)
Comment (MChecaH) [reply wiki/Guides/Common_modding_mistakes/es.md:77]: For this one, [I based myself on how the wiki translated Storyboard to](https://osu.ppy.sh/wiki/es/Storyboard), though that was only done on the title. Unsure whether that was intended, though I'll apply this one being what's most commonly referred to regardless of the language.

File: wiki/Guides/Common_modding_mistakes/es.md (hunk 4 above)
Comment (MChecaH) [reply wiki/Guides/Common_modding_mistakes/es.md:9]: [Ho sento.](https://esadir.cat/gramatica/criteris/diacritics)

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar iendo por las ramas, olvidándote de mejorar el mapa en el proceso.
+
+---
+
+```md
+El soporte para la resolución panorámica es inconsistente y tu mapa no tiene guin gráfico, por lo tanto desactívalo para todas las dificultades.
+00:00:000 - La manta está desalineada por 2 píxeles.
+00:01:000 - El punto de vista previa no está sobrelapado con el divisor de ritmo.
+00:02:000 - Mueve 1px arriba.
```
[the remark this answers is not in the document]
Comment (MChecaH) [reply wiki/Guides/Common_modding_mistakes/es.md:80]:
```suggestion
00:02:000 - Muévelo hacia arriba por 1 px.
```

Added the missing accent. Will apply.

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+Los siguientes puntos están presentados de manera que un mal ejemplo de modding se muestra arriba, y una explicación de por qué es un error abajo.
+
+Aunque los ejemplos enseñados son modelados a partir de contexto de mapping específico a [osu!](/wiki/Game_mode/osu!), también se pueden aplicar a otros [modos de juego](/wiki/Game_mode).
+
+---
+
+```md
+00:00:000 - Borra este slider
+00:01:000 - Añade círculo.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:15]:
```suggestion
00:01:000 - Añade un círculo.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+00:02:000 - Reduce el espacio aquí.
+00:02:000 - Haz que fluya mejor.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:17]:
```suggestion
00:02:000 - Haz que este flow sea más agudo.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+00:04:000 - Ctrl+G
+00:05:000 - Mueve a (100:200)
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:19]:
```suggestion
00:05:000 - Muévelo a (100:200)
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+En general, las notas deberían estar mapeadas a distintos sonidos de la canción para poder reflejarla con exactitud. Mantener la densidad del ritmo con la intensidad de la canción es /.../
+```
+
+**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar iendo por las ramas, olvidándote de mejorar el mapa en el proceso.
+
+---
+
+```md
+El soporte para la resolución panorámica es inconsistente y tu mapa no tiene storyboard, por lo tanto desactívalo para todas las dificultades.
+00:00:000 - El blanket está desalineada por 2 píxeles.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:78]: My fault
```suggestion
00:00:000 - El blanket está desalineado por 2 píxeles.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+00:01:000 - El punto de vista previa no está sobrelapado con el divisor de ritmo.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:79]:
```suggestion
00:01:000 - El punto de vista previa no está ajustado, podría ajustarse.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+Este mod me ha llevado 10 horas de completar, ya puedes estar aplicando mis sugerencias.
+Tu estilo de mapping es muy malo; limítate a mapear como todos los otros.
+00:00:000 - ¿Por qué hay sliders gigantes por todas partes? Colócalos como es debido.
+```
+
+**El objetivo de modear es mejorar el mapa.** No debes forzar tu opinión al mapper. Si deciden rechazar varias sugerencias que crees que mejorarían el mapa, eso es su problema. En estos casos, acéptalo y sigue con tus cosas, pero considera su razonamiento e intenta predecir esto en futuros casos.
+
+```md
+00:00:000 - Este ritmo está mal.
+00:01:000 - Pienso que puedes mejorar este arte de sliders un poco, intenta usar una figura más estética.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:105]:
```suggestion
00:01:000 - Pienso que puedes mejorar este arte de los sliders un poco, intenta usar una figura más estética.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+00:02:000 - Deberías de probar con algo más complejo aquí, esto se siente muy simple para lo que la canción sugiere.
+00:03:000 - El flujo está mal aquí.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:107]:
```suggestion
00:03:000 - El flow está mal aquí.
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+**Si de verdad crees que el mapper debería de rehacer el mapa, explica que está mal y por qué.** No tiene ningún sentido en rehacerlo si no se aprende nada de ello, asi que asegúrate de mencionarlo en tu mod. Recuerda de poner ejemplos!
+
+**Los remapeos deberían de ser el último recurso.** Evita sugerirlos a menos que el concepto del mapa esté fundamentalmente roto, y que remapear supondría un esfuerzo menor a construir sobre lo que ya está hecho.
+
+---
+
+```md
+Mi mod es de mejor calidad porque tengo más experiencia que tú, debes aplicarlo de cualquier modo!
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:125]:
```suggestion
Mi mod es de mejor calidad porque tengo más experiencia que tú, ¡debes aplicarlo de cualquier modo!
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+No sabes lo que es un mapa de calidad, aplica mi mod porque yo sí que lo sé.
+```
+
+**Que un mod sea de calidad o no depende del mapper.** Solo mirando el mod no se puede saber. Para saberlo, debes de ponerte desde la perspectiva del mapper. ¿Aplicarían estas sugerencias? ¿De verdad que són útiles? ¿Se entenderá el razonamiento?
+
+---
+
+```md
+00:00:000 - has que esto fluya bien
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:134]:
```suggestion
00:00:000 - haz de esto un buen flow
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+00:01:000 - esto puede mejorar
+00:02:000 - mala estructura
+00:03:000 - ritmo está mal
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:137]:
```suggestion
00:03:000 - el ritmo está mal
```

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+```md
+La estética es subjetiva asi que voy a ignorar eso.
+```
+
+**Lo que crees que es objetivo para ti sigue siendo en parte subjetivo hasta cierto punto.** Es mejor evitar pensar que la subjetividad es blanca o negra, y mencionar cualquier cosa que pueda mejorar el mapa.
+
+---
+
+```md
+00:00:000 - tio, ¿siempre mapeas así de mal?
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:163]:
```suggestion
00:00:000 - tío, ¿siempre mapeas así de mal?
```

File: wiki/Guides/Common_modding_mistakes/es.md (hunk 9 above)
Comment (MChecaH) [reply wiki/Guides/Common_modding_mistakes/es.md:78]: Went over my head as well.

File: wiki/Guides/Common_modding_mistakes/es.md (hunk 10 above)
Comment (MChecaH) [reply wiki/Guides/Common_modding_mistakes/es.md:79]: Didn't find better words back then. Sounds way better.

File: wiki/Guides/Common_modding_mistakes/es.md [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+
+**Si algo parece estar por todo el mapa, puede que haya sido intencional.** En estos casos, o bien explica en detalle por qué crees que está mal, o asegúrate de que aparecen en partes similares de la canción. Si són inconsistentes y no parece haber patrón alguno, estaría bien preguntar por ello.
+
+---
+
+```md
+En general, las notas deberían estar mapeadas a distintos sonidos de la canción para poder reflejarla con exactitud. Mantener la densidad del ritmo con la intensidad de la canción es /.../
+```
+
+**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar iendo por las ramas, olvidándote de mejorar el mapa en el proceso.
```
Comment (agatemosu) [inline wiki/Guides/Common_modding_mistakes/es.md:72]:
```suggestion
**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar yendo por las ramas, olvidándote de mejorar el mapa en el proceso.
```

**Comment by agatemosu** [comment]
No need to merge master branch

**Comment by agatemosu** [comment]
If you are going to translate other articles, create another branch.

**Comment by MChecaH** [comment]
I am aware of it, I just did not realize this was not a patch branch to begin with. Trying to fix this mishap.

**Comment by MChecaH** [comment]
Fixed that. Sorry about that, not all that used to the flow of things yet.

**Review by TicClick — APPROVED** [review]

**Pull request merged**